test/boulder-tools/requirements.txt

```diff
@@ -1,5 +1,5 @@
-acme>=2.0
-cryptography>=0.7
-PyOpenSSL
-requests
-codespell
+acme==2.6.0
+cryptography==3.4.8
```

Comment:
Why these specific versions? This version of [cryptography](https://pypi.org/project/cryptography/#history) is several releases behind, and same with [PyOpenSSL](https://pypi.org/project/pyOpenSSL/#history).

Maybe instead just make codespell explicit? Then again, this is just for integration testing. 🤷🏼‍♂️

Reply:
wow, huh! I just did `pip install` locally and copied what it output.

yeah, I'll just do codespell for now since it's the only one causing problems.

Reply:
Weiiirdddd